plugins/MumblePluginProcess.hpp

```diff
@@ -0,0 +1,313 @@
+// Copyright 2020 The Mumble Developers. All rights reserved.
+// Use of this source code is governed by a BSD-style license
+// that can be found in the LICENSE file at the root of the
+// Mumble source tree or at <https://www.mumble.info/LICENSE>.
+
+#ifndef MUMBLEPLUGINPROCESS_HPP_
```

Comment:
Why is this a `.hpp`file while we are using `.h` almost everywhere else for our header files?

Reply:
Just because it contains the implementation as well.

However, since you suggested to move it to `.cpp` files, I will rename it to `.h`.

One could argue that `.hpp` would be recommended to make it clear that the header is not compatible with C, but as you said we're already using `.h` everywhere.

Reply:
> Just because it contains the implementation as well.

I think I never saw this kind of reasoning before :thinking: 

I only know that `.hpp` files should be for cpp headers while `.h` files should be for C headers. But I see `.h` being used for cpp as well more often than not and since we're doing it everywhere else already, I'd opt for sticking to this style :)